<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseProtoWidget</class>
 <widget class="QWidget" name="ChooseProtoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>164</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="cellTypeLabel">
     <property name="text">
      <string>Cell Type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cellType"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="protocolType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Simulation Protocol Type</string>
     </property>
     <layout class="QVBoxLayout" name="groupBoxLayout">
      <item>
       <widget class="QRadioButton" name="currentClamp">
        <property name="toolTip">
         <string>Apply current stimulus</string>
        </property>
        <property name="text">
         <string>Current C&amp;lamp - Single Cell</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="voltageClamp">
        <property name="toolTip">
         <string>Clamp cell membrane to specified voltage</string>
        </property>
        <property name="text">
         <string>&amp;Voltage Clamp - Single Cell</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="grid">
        <property name="toolTip">
         <string>Multicellular simulation</string>
        </property>
        <property name="text">
         <string>&amp;2D Grid Protocol</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QPushButton" name="readSettings">
     <property name="text">
      <string>Load Settings</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
